manifests/s/SimonG4ELI/SDR-Radio/3.3-b3195/SimonG4ELI.SDR-Radio.locale.en-US.yaml

```
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: SimonG4ELI.SDR-Radio
+PackageVersion: 3.3-b3195
+PackageLocale: en-US
+Publisher: Simon G4ELI
+PublisherUrl: https://github.com/SimonG4ELI
```

```suggestion
PublisherUrl: https://www.dit-dit-dit.com
```

Personal website is a slightly better "publisher url" since SDR-Radio is not Open Source and there is nothing on Simon's GitHub.